### PR TITLE
Bugfix: Offline bundles should be removed from worker_run

### LINKED ISSUE #876: Offline bundles should be removed from worker_run

### PR DESCRIPTION
**Pull request by fabeschan**
Fixes #876

### CODE REVIEW

**Review by percyliang — COMMENTED** [review]
Can you give more context about what problem this fixes?

**Comment by fabeschan** [comment]
Previously, the only way an offline row in worker_run can be removed is when the worker that is assigned to it is removed (as a dead worker).

If the worker never gets detected as 'dead' by the server for whatever reason then the offline row stays in worker_run forever.

Also there was a bug in resume_bundle() where the last_updated timestamp is not updated if the bundle is in RUNNING state. This field should be regularly updated so that last_updated doesn't fall too far behind and get moved to WORKER_OFFLINE

**Review by percyliang — APPROVED** [review]

File: codalab/model/bundle_model.py [hunk 1]
```diff
@@ -628,6 +629,10 @@ def set_offline_bundle(self, bundle):
                 # The user deleted the bundle.
                 return False
 
+            # Delete row in worker_run
```
Comment (percyliang) [inline codalab/model/bundle_model.py:632]: Is it documented somewhere what the semantics of something being in worker_run is?  This seems like tricky logic, and would be good to document how one should reason about the flow.

Comment (fabeschan) [reply codalab/model/bundle_model.py:632]: I don't recall seeing anything detailing worker_run

and to be honest I'm only making (hopefully educated) guesses to guide my changes

**Pull request merged**